### PR TITLE
NOISSUE - Update kubernetes.md

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

File: docs/kubernetes.md [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
-Add stable chart repository:
+### Stable Helm repository
+Add a stable chart repository:
 ```bash
 helm repo add stable https://kubernetes-charts.storage.googleapis.com/
 ```
 
-## Deploying Mainflux
+### Nginx Ingress Conntroller
+The goal of Ingress controller is the assembly of a configuration file (nginx.conf). The main implication of this requirement is the need to reload NGINX after any change in the configuration file. Follow [this instructions](https://kubernetes.github.io/ingress-nginx/deploy/) to install it.
```
Comment (blokovi) [inline docs/kubernetes.md]: This whole configuration part is confusing, I would just leave last sentence with install instruction.

**Review by blokovi — DISMISSED** [review]

File: docs/kubernetes.md [hunk 2]
```diff
@@ -1,36 +1,34 @@
...
+### kubectl
+Kubectl is official Kubernetes command line client. Follow [this instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/) to install it.
 
-Installing Helm v3 on Linux:
-```bash
-curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 > get_helm.sh
-chmod 700 get_helm.sh
-./get_helm.sh
-```
+Regarding the cluster control with `kubectl`, default config `.yaml` file should be `~/.kube/config`, so if you have another one (for example one downloaded from DO), it should be renamed.
```
Comment (drasko) [inline docs/kubernetes.md]: Why renamed - can not it be pointed to with a CLI switch?

File: docs/kubernetes.md [hunk 3]
```diff
@@ -1,36 +1,34 @@
...
-Add stable chart repository:
+### Stable Helm repository
+Add a stable chart repository:
 ```bash
 helm repo add stable https://kubernetes-charts.storage.googleapis.com/
 ```
 
-## Deploying Mainflux
+### Nginx Ingress Conntroller
+Follow [this instructions](https://kubernetes.github.io/ingress-nginx/deploy/) to install it.
```
Comment (drasko) [inline docs/kubernetes.md]: `these instructions`

File: docs/kubernetes.md (hunk 2 above)
Comment (drasko) [reply docs/kubernetes.md]: Maybe drop this part of the sentence `, so if you have another one (for example one downloaded from DO), it should be renamed.` - looks redundant, and it is explanation of general K8s stuff

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**